doc/modules/ROOT/pages/index.adoc

```diff
@@ -24,6 +36,28 @@ include::{templates-url}/microservices/prerequisites.adoc[]
 
 Before starting to use Hazelcast inside the application, let's configure it in `HazelcastApplication.java`:
 
+[tabs]
+====
+Client::
++
+--
+NOTE: You need to have a running Hazelcast before using client.
```

Comment:
I think we should maybe write how to start Hazelcast server. Otherwise, the readers need to figure it out by themselves.